### PR TITLE
Forgot to apply change in base URL policy globally.

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

File: coastal-hazards-portal/src/main/resources/example_context.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@ application services -->
 coastal-hazards.base.url: This is only necessary on public facing URLs. The 
 Jersey framework seems to clobber contextPath in the request object. 
 Therefore, a quick fix is to use this value, set on public facing servers (qa/prod) 
-in order to get the public mapping of where the application is located. -->
+in order to get the public mapping of where the application is located. CANNOT have a training slash. -->
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/resources/example_context.xml]: Got to train you how to spell :D

File: coastal-hazards-portal/src/main/webapp/OpenID/oid-login.jsp [hunk 2]
```diff
@@ -14,7 +14,8 @@
 %>
 <%
     String originatingUri = StringUtils.isNotBlank(request.getParameter("originating_uri")) ? request.getParameter("originating_uri") : "";
-	String baseUrl = StringUtils.isNotBlank(request.getContextPath()) ? request.getContextPath() : props.getProperty("coastal-hazards.base.url");
+	String baseUrl = props.getProperty("coastal-hazards.base.url");
+	baseUrl = StringUtils.isNotBlank(baseUrl) ? baseUrl : request.getContextPath();
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/OpenID/oid-login.jsp]: Couldn't you just reverse my one-liner to keep it as a one liner?

File: coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/publish/item/index.jsp [hunk 3]
```diff
@@ -39,7 +39,8 @@
 	String vFontAwesome = getProp("version.fontawesome");
 	String vOpenlayers = getProp("version.openlayers");
 	String vSugarJs = getProp("version.sugarjs");
-	String baseUrl = StringUtils.isNotBlank(request.getContextPath()) ? request.getContextPath() : getProp("coastal-hazards.base.url");
+	String baseUrl = props.getProperty("coastal-hazards.base.url");
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/publish/item/index.jsp]: This seems to be used a lot. We should probably make a util jsp to include in these jsps maybe?

File: coastal-hazards-portal/src/main/resources/example_context.xml (hunk 1 above)
Comment (thongsav-usgs) [inline coastal-hazards-portal/src/main/resources/example_context.xml]: lulz

File: coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/publish/item/index.jsp (hunk 3 above)
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/publish/item/index.jsp]: I still think we should get rid of it all together

File: coastal-hazards-portal/src/main/webapp/OpenID/oid-login.jsp (hunk 2 above)
Comment (thongsav-usgs) [inline coastal-hazards-portal/src/main/webapp/OpenID/oid-login.jsp]: one liner would force me to call getProperty twice, i think string allocation is cheaper

File: coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/publish/item/index.jsp (hunk 3 above)
Comment (thongsav-usgs) [inline coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/publish/item/index.jsp]: Yeah, I had the same thought. Will be done.

File: coastal-hazards-portal/src/main/webapp/OpenID/oid-login.jsp (hunk 2 above)
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/OpenID/oid-login.jsp]: Sounds plausible

**Pull request merged**